### PR TITLE
Add support for .jpg and .gif images.

### PR DESCRIPTION
**Pull request by l3iodeez**
Image request logic was abstracted out into a method that takes a file_type as an argument, and paths for the various file types call those methods.

### CODE REVIEW

**Comment by afeld** [comment]
Closes #2.

This is great! Mind adding a note to the README? /cc #9

File: Gemfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 gem 'sinatra'
 
-gem 'rmagick'
+gem 'rmagick', '~> 2.15.4'
```
Comment (afeld) [inline Gemfile]: Is there something special about this version? If so, mind adding a comment? If not, I prefer to keep the version restriction out of the Gemfile.

File: Gemfile.lock [hunk 2]
```diff
@@ -4,7 +4,7 @@ GEM
     rack (1.5.2)
     rack-protection (1.5.3)
       rack
-    rmagick (2.13.3)
+    rmagick (2.15.4)
```
Comment (afeld) [inline Gemfile.lock]: This seems to fix #11! :boom:

**Comment by afeld** [comment]
FYI, best practice would be to do a dependency upgrade separate from implementing a feature, unless the two are connected. If that's the case here, never mind!

**Comment by l3iodeez** [comment]
Sorry about that. I had to change the dependency to get it to run in my environment for some reason (2.13.3 wouldn't install), then I promptly forgot about it.

**Comment by l3iodeez** [comment]
I removed the dependency change and added to the readme.

**Comment by afeld** [comment]
Closes #9.

> Sorry about that. I had to change the dependency to get it to run in my environment

No worries at all. I actually do want to do the upgrade whether it's in this pull request or a different one.

> I removed the dependency change

Mind resetting the Gemfile.lock back to what it was, or re-adding the change to the Gemfile? Need them to be in sync, whether we do the RMagick upgrade in this pull request or a separate one.

**Comment by l3iodeez** [comment]
I re-added the change to the Gemfile. Let me know if any other changes are required.

**Comment by afeld** [comment]
Thanks!

**Comment by afeld** [comment]
Sorry, realizing I confused/contradicted myself. Removed the version restriction in ffc43b279b9036d4eae9fe927e0d693ef260ae81.

**Pull request merged**